wremnants/theory_tools.py

```diff
@@ -240,8 +246,7 @@ def define_central_pdf_weight(df, dataset_name, pdf):
     return df.Define("central_pdf_weight", f"std::clamp<float>({pdfBranch}[0], -theory_weight_truncate, theory_weight_truncate)")
 
 def define_theory_weights_and_corrs(df, dataset_name, helpers, args):
-    if "prefsrLeps" not in df.GetColumnNames():
-        df = define_prefsr_vars(df)
+    df = define_prefsr_vars(df)
```

Comment:
Why get rid of this check? I assume there was some motivation to avoid calling this twice in the past